ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2154,6 +2154,15 @@ router_ospf:
           summary_lsa:
             override_metric: < Integer 1-16777215 >
       mpls_ldp_sync_default: < true | false >
+      # area specific configuration
+      area_specific_config:
+        - ospf_area: < ospf_area >
+          stub: < no_summary | true | on >
```

Comment:
To be consistent with the rest of our data model, we should avoid mixed type variables. Also "on" is not used elsewhere.
I suggest:
```yaml
areas:
  < ospf_area >:
    type: < normal | stub | nssa | default -> normal >
    no_summary: < true | false >
    nssa_only: <true | false >
```

Reply:
done